app/editor/src/features/admin/actions/ActionList.tsx

```diff
@@ -2,7 +2,7 @@ import { FormPage, IconButton } from 'components/form';
 import { IActionModel } from 'hooks/api-editor';
 import React from 'react';
 import { useNavigate } from 'react-router-dom';
-import { useActions } from 'store/hooks/admin/actions';
+import { useActions } from 'store/hooks/admin';
```

Comment:
I reorganized the location of these hooks to reduce boilerplate code where it makes sense.